include/spdlog/common.h

```diff
@@ -173,12 +173,18 @@ using format_string_t = fmt::format_string<Args...>;
 template<class T>
 using remove_cvref_t = typename std::remove_cv<typename std::remove_reference<T>::type>::type;
 
+#if FMT_VERSION >= 90101
+# define FMT_STRING_RUNTIME fmt::runtime_format_string<Char>
+#else
+# define FMT_STRING_RUNTIME fmt::basic_runtime<Char>
+#endif
```

Comment:
I think its better than macro
```C++
template <typename Char>
#if FMT_VERSION >= 90101
using fmt_runtime_string = fmt::runtime_format_string<Char>;
#else
using fmt_runtime_string = fmt::basic_runtime<Char>;
#endif
```

Reply:
I agree. Seems cleaner using `using`. @H1X4Dev Please update the pr